frontend/components/README.md

```diff
@@ -0,0 +1,89 @@
+# MarkBind Vue components
+
+This folder contains MarkBind's [vuejs](https://vuejs.org/) UI components.
+
+
+Most of the styles are based on Bootstrap's markup and CSS. As a result no dependency Bootstrap's JavaScript is required. The only required dependencies are:
+
+* [Vue.js](http://vuejs.org/) (required ^v2.x.x, test with v2.5.16).	They were modified to suit MarkBind's needs for educational and documentation websites.
+* [Bootstrap CSS](http://getbootstrap.com/) (required 4.x.x, test with 4.1.1). VueStrap doesn't depend on a very precise version of Bootstrap.
+
+
+## Components / directives
+
+Many of the components under this folder were originally from [MarkBind's forked project](https://github.com/MarkBind/vue-strap) of [VueStrap](https://github.com/yuche/vue-strap), and were modified to suit MarkBind's needs for educational and documentation websites.
+
+
+Some custom directives were also added for MarkBind's own use.
+
+
+### MarkBind components ported from [Markbind/vue-strap](https://github.com/MarkBind/vue-strap):
+
+- Pic.vue
+- Question.vue
+- Retriever.vue
+- Searchbar.vue
+- SearchbarPageItem.vue
+- Thumbnail.vue
+- TipBox.vue
+
+
+### Custom directives ported from [Markbind/vue-strap](https://github.com/MarkBind/vue-strap)
+
+- Closeable.js
+- Float.js
+
+
+### VueStrap components modified for use in MarkBind; ported from [Markbind/vue-strap](https://github.com/MarkBind/vue-strap), a fork of [yuche/vue-strap](https://github.com/yuche/vue-strap):
+
+- Affix.vue
+- Dropdown.vue
+- Navbar.vue
+- NestedPanel.vue
+- MinimalPanel.vue
+- Tab.vue
+- TabGroup.vue
+- Tabset.vue
+- Typeahead.vue
+
+
+## Installation
+
+### Browser globals
+The `dist` folder contains `vue-strap.js` and `vue-strap.min.js` with all components exported in the <code>window.VueStrap</code> object.
+
+```html	- Pic.vue
+<script src="path/to/vue.js"></script>
+<script src="path/to/vue-strap.js"></script>
+<script>
+    var vm = new Vue({
+        components: {
+            alert: VueStrap.alert
+        },
+        el: "#app",
+        data: {
+            showRight: false,
+            showTop: false
+        }
+    })
+</script>
+```
+
+
+## Docs
+See the [documentation](https://markbind.org/userGuide/usingComponents.html) with live editable examples.	
+
+
+## Local Setup
+* Install with `npm install`.
+* Build with `npm run build`.	
+
+
+## Lint the code for any code and style errors using ESLint
+* For Unix: `npm run lint`	
```

Comment:
Do check the diff when committing: https://github.com/MarkBind/markbind/commit/949d9097eba63b37fdb1e7ab721c9ad5778f380c

```suggestion
* For Unix: `npm run lint`
```